fast-markdown-mcp/src/fast_markdown_mcp/server.py

```
@@ -5,17 +5,19 @@
 import json
 import re
 import asyncio
+import math
```

Ruff (0.8.2)

`math` imported but unused

Remove unused import: `math`

(F401)